core/templates/services/math-interactions.service.spec.ts

```diff
@@ -205,6 +214,77 @@ describe('MathInteractionsService', () => {
     expect(mathInteractionsService.getWarningText()).toBe(
       'Your answer contains an invalid term: 3.4.5');
 
+    expect(mathInteractionsService.validateAlgebraicExpression(
+      'a4', ['a'])).toBeFalse();
+    expect(mathInteractionsService.getWarningText()).toBe(
+      'Your answer contains invalid variable multiplication: a4. ' +
```

Comment:
This seems ungrammatical -- shouldn't it be "an invalid"? 

Also this first part sounds really jargon-y. Perhaps simpler would be: "When multiplying, the variable should come *after* the number: 4a. Please update your answer and try again." (or similar)

What do you think? Might need discussion with @srushtirk and an update to the PRD if we decide to simplify this.

Reply:
I think that sounds better. I've updated it.